README.md

```diff
@@ -0,0 +1,223 @@
+# Centreon Collect                                                                 
+                                                                                   
+Centreon Collect is collection of softwares:                                                                                     
+  * Centreon Engine                                                                  
+  * Centreon Broker                                                                  
+  * Centreon Connectors                                                              
```

Comment:
Make some links to the sections
Centreon Engine => section "Centreon Engine"
etc...

Reply:
Centreon Collect brings also "functional tests" on these softwares.

"functional tests" should point to the section "Centreon tests".